### ISSUE TITLE
sys-audio: official documentation and salt improvement

### ISSUE THREAD

**Issue by neowutran** [QubesOS/qubes-issues#8093]
[How to file a helpful issue](https://www.qubes-os.org/doc/issue-tracking/)

### The problem you're addressing (if any)

Currently there is no official documentation on how to setup "sys-audio".
QubesOS provide a salt to create "sys-audio" but things are missing to make it work. 

The most up to date documentation on how to setup "sys-audio" are splitted across multiple post in the qubes forum
(
https://forum.qubes-os.org/t/setting-up-an-audio-vm/5491/67
https://forum.qubes-os.org/t/proper-guide-on-how-to-use-sys-audio-audiovm-must-be-created/16851/8
https://forum.qubes-os.org/t/debian-minimal-template-for-sys-audio/11108/74
https://forum.qubes-os.org/t/guide-on-using-an-usb-audio-device-with-qubes-4-1/10748/3
)

### The solution you'd like

- Official documentation for sys-audio 
- An update to the sys-audio salt

I could help, at least for the documentation part, if there is no WIP on that subject. 

### The value to a user, and who that user might be

Make it easier to properly setup a sys-audio

**Comment by andrewdavidwong** [comment]
Please note that [every issue must be about a single, actionable thing](https://www.qubes-os.org/doc/issue-tracking/#every-issue-must-be-about-a-single-actionable-thing). Therefore, I'm modifying the title of this issue to be about documentation only. If you wish to open a separate issue for the Salt improvement part, you are free to do s.

**Comment by jamke** [comment]
I agree that documentation for `sys-audio` should exist.
People are not able to use bluetooth headphones with Qubes OS laptops otherwise.

_Related link:_
**Proper guide on how to use sys-audio (AudioVM) must be created**
https://forum.qubes-os.org/t/proper-guide-on-how-to-use-sys-audio-audiovm-must-be-created/16851

**Comment by neowutran** [comment]
thanks, 
started to work on it here: https://github.com/neowutran/qubes-doc/blob/sys-audio/user/advanced-topics/audio-qubes.md
once i think it is ready I will send it on the forum and then do a pull request

**Comment by jamke** [comment]
> started to work on it here: https://github.com/neowutran/qubes-doc/blob/sys-audio/user/advanced-topics/audio-qubes.md
> once i think it is ready I will send it on the forum and then do a pull request
I see no screenshots, is it intentional (not finished) or something else?

**Comment by neowutran** [comment]
> > started to work on it here: https://github.com/neowutran/qubes-doc/blob/sys-audio/user/advanced-topics/audio-qubes.md
> > once i think it is ready I will send it on the forum and then do a pull request
> 
> I see no screenshots, is it intentional (not finished) or something else?

Screenshots are in this repo https://github.com/neowutran/qubes-attachment/tree/audiovm/doc ( for integration with qubes os doc ).  
And before doing a pull request for the documentation, I at least need to properly fix this issue: https://github.com/QubesOS/qubes-issues/issues/8109 ( or else the permission configuration I mentionned in the WIP doc won't be enough )

**Comment by GammaSQ** [comment]
@neowutran I'm so sorry, I didn't realize someone was already preparing a PR. I've submitted mine yesterday, but it's more of a list of notes I kept while setting up my audio-VM. It's more focused on minimal setup and could use another pair of eyes, especially considering your background knowledge. Have a look here: https://github.com/QubesOS/qubes-doc/pull/1366

**Comment by neowutran** [comment]
> @neowutran I'm so sorry, I didn't realize someone was already preparing a PR. I've submitted mine yesterday, but it's more of a list of notes I kept while setting up my audio-VM. It's more focused on minimal setup and could use another pair of eyes, especially considering your background knowledge. Have a look here: [QubesOS/qubes-doc#1366](https://github.com/QubesOS/qubes-doc/pull/1366)

My personal most uptodate doc is in the forum ( https://forum.qubes-os.org/t/audio-qube/20685  ) 

Some comments: 

"Qubes 4.2 switched from pipewire to pulseaudio.", pretty sure it is the other way, from pulseaudio to pipewire

"The volume-tray-icon doesn't work for audio-VMs at the moment.", I never tried to have a system tray for audio, but probably better to fix it instead of documenting it ( https://forum.qubes-os.org/t/audio-qube/20685/64 )

 --------------

The things that annoy me is the "policy" part. It probably better to fix it before an official doc

**Comment by jamke** [comment]
Can having `sys-audio` be included in the roadmap? E.g. for R4.4? 

It seems to be such a demanded feature among users, that even somewhat increases security, but still is half-supported and not presented officially, unlike `sys-usb`.

And thanks to @neowutran for keeping the topic and guides alive on the forum.
